extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/attribute/AllRequestHeadersAttribute.java

```diff
@@ -1,36 +1,94 @@
 package io.quarkus.vertx.http.runtime.attribute;
 
+import java.util.List;
 import java.util.Map;
 import java.util.StringJoiner;
 
+import org.eclipse.microprofile.config.ConfigProvider;
+
 import io.vertx.core.MultiMap;
+import io.vertx.core.http.HttpHeaders;
 import io.vertx.ext.web.RoutingContext;
 
 public class AllRequestHeadersAttribute implements ExchangeAttribute {
 
     public static final AllRequestHeadersAttribute INSTANCE = new AllRequestHeadersAttribute();
 
-    private AllRequestHeadersAttribute() {
+    private static final String AUTHORIZATION_HEADER = String.valueOf(HttpHeaders.AUTHORIZATION);
+    private static final String COOKIE_HEADER = String.valueOf(HttpHeaders.COOKIE);
+    private static final String MASKED_HEADERS_PROPERTY = "quarkus.http.access-log.masked-headers";
+    private static final String MASKED_COOKIES_PROPERTY = "quarkus.http.access-log.masked-cookies";
 
+    private final List<String> maskedHeaders;
+    private final List<String> maskedCookies;
+
+    private AllRequestHeadersAttribute() {
+        maskedHeaders = ConfigProvider.getConfig().getOptionalValues(MASKED_HEADERS_PROPERTY, String.class).orElse(List.of());
+        maskedCookies = ConfigProvider.getConfig().getOptionalValues(MASKED_COOKIES_PROPERTY, String.class).orElse(List.of());
     }
 
     @Override
     public String readAttribute(RoutingContext exchange) {
-        final MultiMap headers = exchange.request().headers();
+        return readAttribute(exchange.request().headers());
+    }
 
+    String readAttribute(MultiMap headers) {
         if (headers.isEmpty()) {
             return null;
         } else {
             final StringJoiner joiner = new StringJoiner(System.lineSeparator());
 
             for (Map.Entry<String, String> header : headers) {
-                joiner.add(header.getKey() + ": " + header.getValue());
+                joiner.add(header.getKey() + ": " + maskHeaderValue(header.getKey(), header.getValue()));
             }
 
             return joiner.toString();
         }
     }
 
+    String maskHeaderValue(String headerName, String headerValue) {
+        if (headerValue == null) {
+            return null;
+        }
+
+        if (AUTHORIZATION_HEADER.equalsIgnoreCase(headerName)) {
+            return maskAuthorizationHeaderValue(headerValue);
+        }
+
+        if (COOKIE_HEADER.equalsIgnoreCase(headerName)) {
+            return maskCookieHeaderValue(headerValue);
+        }
+
+        if (maskedHeaders.contains(headerName)) {
+            return "...";
```

Comment:
```suggestion
            return "***";
```

nitpick: Wouldn't this be better? 

Intuitively, I'm considering '...' as a 'continue of some long value'. Using the '***' as a mask seems more appropriate for this as it represents some sensitive information.

@sberyozkin WDYT?